### PR TITLE
chore: bump to `internal@1.0.4`

### PR DESCRIPTION
**Pull request by iuioiua**

### CODE REVIEW

**Comment by kt3k** [comment]
Is this for making this work? https://github.com/denoland/std/commit/6a4eb6cb9185b0b9e0816a227c04b5bf039b99ef

**Comment by iuioiua** [comment]
Yes. Because `@std/internal/assertion-state` was added but never published.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -49,9 +49,8 @@ jobs:
           TZ=America/Toronto deno test datetime
         if: matrix.os == 'ubuntu-latest'
 
-      # TODO(kt3k): Enable this test when `deno check` command supports workspace resolution
-      # - name: Type check browser compatible modules
-      #   run: deno task test:browser
+      - name: Type check browser compatible modules
```
Comment (kt3k) [inline .github/workflows/ci.yml:52]: Maybe let's not try enabling this in this PR?

**Review by kt3k — APPROVED** [review]
LGTM

**Pull request merged**